src/tmpfiles/tmpfiles.c

```diff
@@ -1958,6 +1958,11 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
         if (!should_include_path(i.path))
                 return 0;
 
+        if (!needs_glob(i.type) &&
+            chase_symlinks(i.path, NULL, CHASE_NO_AUTOFS, NULL) == -EWOULDBLOCK)
+                /* autofs automount point - not safe to touch */
+                return 0;
```

Comment:
so far, we first parsed everything and only then started applying things. I am pretty sure we should do this check as part of the second step too. After all, an earlier tmpfiles rule might create a symlink a follow-up rule then relies on to reference an autofs mount, and we should detect that too. Hence I am pretty sure we should read all entries here, and then filter out the autofs rules as late as possible.